brigade/plugins/tasks/networking/netmiko_send_config.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from brigade.core.task import Result
+
+
+def netmiko_send_config(task, config_commands=None, config_file=None, **kwargs):
+    """
+    Execute Netmiko send_config_set method (or send_config_from_file)
+    Arguments:
+        config_commands(list, optional): Commands to configure on the remote network device.
+        config_file(str, optional): File to read configuration commands from.
+
```

Comment:
Probably remove the space, not sure if sphinx will interpret it as a new section